clang/test/Preprocessor/embed_file_not_found_chevron.c

```diff
@@ -1,4 +1,5 @@
-// RUN: %clang_cc1 -std=c23 %s -E -verify
+// RUN: %clang_cc1 -std=c23 %s -E -verify --embed-dir=%S --embed-dir=%S/Inputs
 
+#embed <jk.txt>
```

Comment:
It feels like this test is unrelated to the test name now; should we move this into its own test file named something like `embed_search_paths.c` or something?

Reply:
Makes sense.